packages/amplify_analytics_pinpoint/lib/method_channel_amplify.dart

```diff
@@ -74,7 +74,7 @@ class AmplifyAnalyticsPinpointMethodChannel extends AmplifyAnalyticsPinpoint {
   }
 
   @override
-  Future<void> unregisterGlobalProperties({List<String> propertyNames}) async {
+  Future<void> unregisterGlobalProperties({List<String>? propertyNames = const []}) async {
```

Comment:
non-blocking question: Why not make this required as unregistering nothing would not be very useful? Is it to minimize upgrading/migrating for customers in the event they have already called this with nothing?

Reply:
hey, if you pass nothing to unregisterGlobalProperties it will unregister all global properties!  This is the behavior of the native APIs that we emulate.

Reply:
oh man! thanks for the important info.